integration_tests/integration/bookedtoday/arrivals/singleExistingRecordFound.spec.ts

```diff
@@ -0,0 +1,149 @@
+import Page from '../../../pages/page'
+import ChoosePrisonerPage from '../../../pages/bookedtoday/choosePrisoner'
+import SingleRecordFoundPage from '../../../pages/bookedtoday/arrivals/singleRecordFound'
+import SexPage from '../../../pages/bookedtoday/arrivals/sexPage'
+import CheckAnswersPage from '../../../pages/bookedtoday/arrivals/checkAnswers'
+import ConfirmAddedToRollPage from '../../../pages/bookedtoday/arrivals/confirmAddedToRoll'
+import Role from '../../../../server/authentication/role'
+import expectedArrivals from '../../../mockApis/responses/expectedArrivals'
+
+const expectedArrival = expectedArrivals.court.notCurrent
+
+context('Confirm Arrival', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn', Role.PRISON_RECEPTION)
+    cy.task('stubPrison', 'MDI')
+    cy.task('stubAuthUser')
+    cy.task('stubUserCaseLoads')
+    cy.task('stubTransfers', { caseLoadId: 'MDI', transfers: [] })
+    cy.task('stubMissingPrisonerImage')
+  })
+
+  it('PER record should contain a full set of correctly formatted move data', () => {
```

Comment:
Currently working to get this test working, as there is a cookie I think we need to implement the complete journey of that cookie for the test to pass - it's currently not redirecting from the sex page to the imprisonment status page which is confusing me as I hardcoded sex as 'MALE' in the cookie therefore the redirect should happen..